cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -181,7 +181,7 @@
 
 {% macro render(controls, index) %}
     <div class="o-filterable-list-controls"
-         id="o-filterable-list-controls">
+         id="o-filterable-list-controls-{{ index | string }}">
```

Comment:
This is going to break the filter that relies on this `id` to load to this section the page after filtering.